docker/docker-compose.yml

```diff
@@ -305,7 +305,12 @@ services:
   redis:
     image: redis:latest
     container_name: 'redis'
-    command: ['redis-server', '--appendonly', 'yes']
+    command:
+      [
+        'bash',
+        '-c',
+        'printf "y\n" | redis-check-aof --fix /data/appendonly.aof ; redis-server --appendonly yes',
```

Comment:
Do we need that `printf "y/n"` part?

Reply:
It seems to be the case, since I ran it without it, and the `redis-check-aof` would prompt for confirmation. I checked the logs to confirm it.

Reply:
I'm a bit confused. I can't reproduce this (`redis-check-aof` asking for confirmation), shouldn't we just print `y` instead of the 2 options `y/n`? I think we could just pass `y`like this `y | redis-check-aof`.